### PR TITLE
Count tasks by task-document not by task historyState

### PR DESCRIPTION
**Pull request by kennsippell**
The current implementation counts the number of elements in the task history - or the number of times tasks have change state. So if a task moves from Draft (in the future) to Ready (now showing) - the current dashboards will show this as two separate tasks (one Draft and one Ready).

The most important thing we do in App Monitoring's tasks tab is to monitor for task explosions and finding erroneous tasks. Task explosions happen when too many task documents are generated - not when tasks frequently change state. 

This change makes it so there is one task event monitored on the documents `authoredOn` date. The state of the task is displayed as that task's last known state.

#50

### CODE REVIEW

**Review by freddieptf — APPROVED** [review]
LGTM! Makes data in the `Task by State` chart less complicated

**Comment by kennsippell** [comment]
@freddieptf Oops. There is a second use of stateHistory in get_tasks_by_user

File: postgres/functions/get_task_by_user.sql [hunk 1]
```diff
@@ -23,27 +23,28 @@ BEGIN
...
+    SELECT 
+        doc->>''user'' as chw,
+        doc#>>''{emission,contact,name}'' as contact,
+        doc#>>''{state}'' as state,
+        (doc#>>''{emission,title}'') as task,  
+        date_trunc(''day'', to_timestamp((doc ->> ''authoredOn'')::bigint / 1000)) AS day
+    FROM 
+        couchdb
+    WHERE 
+        doc ->> ''type'' = ''task''
```
Comment (derickl) [inline postgres/functions/get_task_by_user.sql:43]: I'm curious about the double single-quotes. Are they necessary?

Comment (kennsippell) [reply postgres/functions/get_task_by_user.sql:43]: Weird - right?  This is a query [inside a string](https://github.com/medic/cht-app-monitoring-data-ingest/blob/main/postgres/functions/get_task_by_state.sql#L25) which is why we use double quotes.

**Pull request merged**